### PR TITLE
Jabber (XMPP) notifications

### PR DESCRIPTION
**Pull request by xsuchy09**
*add - Jabber/XMPP notifications @see https://github.com/phpservermon/phpservermon/issues/829
*fix - config template (telegram vs pushover)

### CODE REVIEW

**Comment by xsuchy09** [comment]
More info in the original #829.

**Comment by xsuchy09** [comment]
I set it as new version 3.4.7 because of some database changes.

File: src/includes/psmconfig.inc.php [hunk 1]
```diff
@@ -130,3 +130,15 @@
      */
     define('PSM_MODULE_DEFAULT', 'server_status');
 }
+
+if (defined('PSM_JABBER_FORCE_TLS') === false) {
```
Comment (TimZ99) [inline src/includes/psmconfig.inc.php:134]: So it's always true?

Comment (xsuchy09) [reply src/includes/psmconfig.inc.php:134]: This is just test if user defines this constant with own value (should be just true/false btw). If not we define it with true value as default ...

Comment (TimZ99) [reply src/includes/psmconfig.inc.php:134]: 😅 haha whoops, didn’t look close enough.

**Comment by xsuchy09** [comment]
@TimZ99 pls confirm this. I merged original develop into my fork via Github ... so it has no sign ... then I edit jabber_password input as one of encrypted one as I wrote. And refactor sending messages - login will be just once and sending messages in foreach ...

edit: If you don't want to rebase, we can cancel this part (reset to 
71df5bf) and make new issue for that where I will solve it ...

**Review by TimZ99 — APPROVED** [review]

**Comment by xsuchy09** [comment]
Hi @TimZ99 , pls can you update the docs (http://docs.phpservermonitor.org/en/latest/index.html)? There is no last version with Jabber in the FAQ (https://github.com/phpservermon/phpservermon/blob/develop/docs/faq.rst#how-do-i-setup-jabber-notifications-from-google-account) and others improvements from last commits.

And we should add link to that FAQ part near Jabber configuration directly in the app - for example it is not so easy for google - it is needed to make an app password for example etc. Common user can not know that.

**Comment by TimZ99** [comment]
The docs should update automatically. There was a bug in it, fixed that with 7968e87. Hopefully it will update this time.

**Comment by xsuchy09** [comment]
Ok. Thanks and hope so :+1:

**Pull request merged**